code/datums/components/anti_magic.dm

```diff
@@ -33,12 +33,14 @@
 	UnregisterSignal(user, COMSIG_MOB_RECEIVE_MAGIC)
 
 /datum/component/anti_magic/proc/protect(datum/source, mob/user, _magic, _holy, major, self, list/protection_sources)
-	if(((_magic && magic) || (_holy && holy)) && (self && blocks_self))
+	if(((_magic && magic) || (_holy && holy)) && (!self || !blocks_self))
```

Comment:
:smoking:

Reply:
Fuck there's an extra ! in there, will fix in a bit

Reply:
Fuck there's an extra ! in there, will fix in a bit

Reply:
Fuck there's an extra ! in there, will fix in a bit

Reply:
an extra you say?

Reply:
You saw nothing